Tests/testing-framework.js

```diff
@@ -0,0 +1,7 @@
+const testEquals = (expectedOutput, expectedInput) => expectedInput === expectedOutput;
+const testIncrease = (expectedOutput, expectedInput) => expectedInput > expectedOutput;
+
+module.exports = {
+    testEquals,
+    testIncrease
+};
```

Comment:
Set up of test runner file, shouldn't have pre populated it until I knew what tests I would need to perform.